src/fsharp/MethodOverrides.fs

```diff
@@ -305,12 +305,12 @@ module DispatchSlotChecking =
                         | [] -> 
                             noimpl()
                         | [ Override(_,_,_,(mtps,_),argTys,_,_,_) as overrideBy ] ->
-                            let possibleDispatchSlots =
+                            let notMoreThanOnePossibleDispatchSlot =
                                 dispatchSlots
                                 |> List.filter (fun (RequiredSlot(dispatchSlot,_)) -> IsNameMatch dispatchSlot overrideBy && IsImplMatch g dispatchSlot overrideBy)
-                                |> List.length
+                                |> isNilOrSingleton
                            
-                            if possibleDispatchSlots > 1 then
+                            if not notMoreThanOnePossibleDispatchSlot then
```

Comment:
No!!!! ...  
It took a third look for me to release that this said __not not....__ so a better name for __notMoreThanOnePossibleDispatchSlot__ please.